docs/tools/offline/cli-test-debug.md

```diff
@@ -99,6 +145,12 @@ You can specify to only **build** the tests by using the `--compile` option:
 $ mbed test -m K64F -t GCC_ARM --compile
 ```
 
+In case of icetea the tests itself is not build but the tests requiring application is build
```

Comment:
I would suggest the phrasing to be a bit simpler here: "In case of icetea, only the test applications are built:"

Reply:
agreed